js/java.js

```diff
@@ -0,0 +1,27 @@
+$(document).ready(function() {
+
+  $('#buttonInput').on('submit', function(e) {
+    e.preventDefault();
+      var $addToList = $('#buttonInput input').val();
+      $('#buttonInput input').val('');
+      
+      $('ul').append('<li>' + $addToList + '<span style="color: red;"> X</span></li>');
+        alert("you probably want to add more WDI");
+  });
+
+});
+
+
+$('ul').on('click' , 'li span', function() {
+    $(this).parent().remove();
+  });
+$('span').style
+
+
+
+// Why did the first function only work if "buttonInput" wasn't in <form> tag? Tried with div and wound't run.
+
+// Please explain $(this)
```

Comment:
```$(this)``` is more or less a shortcut for what's being targeted by jQuery. See the below examples, where ```$(this)``` is a placeholder for ```$('span')```